client/src/components/settings/Username.js

```diff
@@ -59,6 +60,7 @@ class UsernameSettings extends Component {
     this.state = {
       isFormPristine: true,
       formValue: props.username,
+      formDisplayValue: props.displayUsername,
```

Comment:
```suggestion
      formDisplayValue: props.displayUsername
        ? props.displayUsername
        : props.username,
```
Needs to fallback to `username` in case `displayUsername` doesn't exist.